cmd/cody-gateway/internal/httpapi/embeddings/handler.go

```diff
@@ -159,6 +159,8 @@ func NewHandler(
 			}
 
 			w.Header().Add("Content-Type", "application/json; charset=utf-8")
+			// Write implicitly returns a 200 status code:
+			resolvedStatusCode = 200
```

Comment:
In case we ever update this earlier without returning early:

```suggestion
			// Write implicitly returns a 200 status code if one isn't set yet
			if resolvedStatusCode <= 0 {
				resolvedStatusCode = 200
			}
```